src/main/java/it/gov/pagopa/payhub/activities/activity/sendnotification/RetrieveNotificationDateActivity.java

```diff
@@ -0,0 +1,21 @@
+package it.gov.pagopa.payhub.activities.activity.sendnotification;
+
+import io.temporal.activity.ActivityInterface;
+import io.temporal.activity.ActivityMethod;
+import it.gov.pagopa.pu.sendnotification.dto.generated.SendNotificationDTO;
+
+/**
+ * Interface to retrieve notification date.
+ */
+@ActivityInterface
+public interface RetrieveNotificationDateActivity {
+
+    /**
+     * Retrieve notification date process.
+     *
+     * @param sendNotificationId the ID of send notification
+     */
+    @ActivityMethod
+    SendNotificationDTO retrieveNotificationDate(String sendNotificationId);
```

Comment:
As suggested on Confluence (https://pagopa.atlassian.net/wiki/spaces/SPAC/pages/1601241160/Notifica+SEND), the proper method name is the following
```suggestion
    SendNotificationDTO sendNotificationDateRetrieve(String sendNotificationId);
```